rs/rust_canisters/call_loop_canister/src/main.rs

```diff
@@ -0,0 +1,63 @@
+use candid::candid_method;
+use ic_cdk::call::Call;
+use ic_cdk_macros::update;
+use ic_principal::Principal;
+use std::future::IntoFuture;
+
+/// Takes the total number of bytes to send in a single message (in megabytes).
+#[candid_method(update)]
+#[update]
+async fn send_calls(megabytes_to_send: u32) {
+    let calls = (0..megabytes_to_send)
+        .map(|i| {
+            let mut slice = [0; 29];
+            slice[..4].copy_from_slice(&i.to_le_bytes());
+            let canister = Principal::from_slice(&slice);
+            Call::unbounded_wait(canister, "")
+                .with_raw_args(&[5; 1024 * 1024])
```

Comment:
nit: Define a const for a MiB to make it a bit more clear.

Reply:
done